instagram/src/components/PostContainer/PostsPage.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react';
+import dummyData from '../../dummy-data';
+import SearchBar from '../SearchBar/SearchBar';
+import Posts from './/Posts';
+
+class PostsPage extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      postData: dummyData,
+      searchString: '',
+    }
+  }
+
+  searchHandler = event => {
```

Comment:
For some reason, it won't pull up fortnite in the search, it works for the other two cards though. I even test it on your netlify as well.

Reply:
Yeah my search was behaving badly but I couldn't figure out what was wrong or how to resolve...

Reply:
So after looking over your code this morning, I finally got the `<Searchbar/>` to update state in your `<Postpage/>`, but the problem I'm having is you having the logic set-up within `<Post/>`. That kinda defeat the purpose of reusability because that component can only fulfill one function, a solution would be creating a handler inside your state component which would be `<Postpage/>`. So that way you can `.filter` through things and adjust state accordingly save it somewhere so the page can render the result from `.filter` method.

Reply:
So the way filter was done on the Sprint Challenge today has opened my eyes to what I should have been doing.